apps/dashboard/app/views/batch_connect/sessions/index.html.erb

```diff
@@ -49,4 +49,5 @@ locals: {
 </div>
 <div id="full-page-spinner" class="d-none">
   <div class="spinner-border" role="status"></div>
-</div>
\ No newline at end of file
+</div>
+<div id="sr-live-region" aria-live="polite" class="visually-hidden"></div>
```

Comment:
This is a good idea to put updates here. So much so that this should likely go to the application.html.erb so it's on every page.